packages/editor-ui/src/composables/usePushConnection.ts

```diff
@@ -199,6 +200,21 @@ export function usePushConnection({ router }: { router: ReturnType<typeof useRou
 				return false;
 			}
 
+			if (receivedData.type === 'executionFinished') {
+				const workflow = workflowsStore.getWorkflowById(receivedData.data.workflowId);
+				const isEasyAIWorkflow =
+					workflow?.meta?.templateId === EASY_AI_WORKFLOW_JSON.meta?.templateId;
```

Comment:
Agree this doesn't look good but, as you mentioned we are pulling this from a constant so It's pretty much in our hands. Also, the first `?` is to make tests work with non-existing `workflow` since there we don't care about it.
But suggestion accepted. Will rework to make it clearer 👍